cli/src/memo.rs

```diff
@@ -0,0 +1,22 @@
+use solana_sdk::instruction::Instruction;
+use solana_sdk::pubkey::Pubkey;
+use std::str::FromStr;
+
+pub trait WithMemo {
+    fn with_memo(self, memo: Option<&String>) -> Self;
+}
+
+impl WithMemo for Vec<Instruction> {
+    fn with_memo(mut self, memo: Option<&String>) -> Self {
```

Comment:
`Option<&String>` - this is a strange type. Can we pass `Option<String>` instead?

Reply:
As per discussion on discord, I rescind this request :)